### PR TITLE
New way to define attribute accessors and mutators in Laravel 8.77:

### PR DESCRIPTION
**Pull request by TechyMS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3762,3 +3763,29 @@ return Http::withOptions([
...
+
+```php
+// Before, two-method approach
+public function setTitleAttribute($value)
+{
+    $this->attributes['title'] = strtolower($value);
+}
+public function getTitleAttribute($value)
+{
+    return strtolower($value);
```
Comment (kang8) [inline README.md:3778]: It is `strtoupper($value)`?

**Review by kang8 — CHANGES_REQUESTED** [review]

**Comment by PovilasKorop** [comment]
@TechyMS we released a new set of tips with another Pull Request, so now there's a conflict. Could you resolve the conflict, and fix the `strtoupper`, and then I will merge.

**Comment by TechyMS** [comment]
Okay

**Comment by PovilasKorop** [comment]
@TechyMS also, I think this tip is not in "Random" but should be in "Eloquent" section. Please move there, and also increase the number of tips - both in Eloquent section and in overall number.

**Comment by TechyMS** [comment]
Sure

**Comment by TechyMS** [comment]
is it ok now?

**Comment by PovilasKorop** [comment]
@TechyMS almost, just need to change one number, I will do it in Master. Thanks for the contribution!

**Pull request merged**